packages/client/src/comp/states/Kansas.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { BaseInput } from '../util/Input'
+import { KansasInfo } from '../../common'
+import { SignatureBase, StatelessInfo, NoSignature } from './Base'
+import { Upload } from '../util/Upload'
+import { TogglableDropdown } from '../util/TogglableDropdown'
+
+const options = [
+  'Driver\'s License Number',
+  'Copy of Photo ID',
+] as const
+
+export const Kansas = () => {
+  const [idPhoto, setIdPhoto] = React.useState<string>()
+  const [idType, setIdType] = React.useState<'Number' | 'Image'>('Number')
```

Comment:
Let's turn `option` into a union type (like `State`) and use those string types directly here.